tools/run_tests/artifacts/build_artifact_python.sh

```diff
@@ -22,8 +22,17 @@ export PYTHON=${PYTHON:-python}
 export PIP=${PIP:-pip}
 export AUDITWHEEL=${AUDITWHEEL:-auditwheel}
 
-# Install Cython to avoid source wheel build failure.
-"${PIP}" install --upgrade cython
+if [ "$GRPC_SKIP_PIP_CYTHON_UPGRADE" == "" ]
```

Comment:
Nit: Why not `if [ ! -z "$GRPC_SKIP_PIP_CYTHON_UPGRADE" ]` ?

Reply:
Mostly for consistency. The `"$variable" == ""` construct is used very commonly in our other .sh scripts and since handling of conditions and their semantics in bash is a non-trivial topic, I just wanted to use what's used in our other scripts to reduce the intellectual load.